tests/markdowns.py

```diff
@@ -0,0 +1,20 @@
+"""
```

Comment:
Do we actually make tests here ? We could simply check the str length to ensure there are no regressions when the code will be changed in the future.

Right now, it looks like it serves the only purpose to increase coverage

Reply:
That's right
I wrote some tests